src/content/structured/components/side-nav/guidance.mdx

```diff
@@ -98,13 +98,13 @@ An example of the side navigation component.
 
 ## When to use
 
-Use side navigation to help provide more screen space for main content. Side navigation is well-suited to apps that have mutually exclusive features which need to be switched between easily on any page.
+Use side navigation to help provide more screen space for main content. Side navigation is well-suited to apps that have mutually exclusive features that need to be switched between easily on any page.
 
 Use side navigation alongside additional components like [page headers](/components/page-header) to create a variety of different [layouts](/styles/layout-spacing) and navigation styles.
 
 ## When not to use
 
-For intranet-style and content heavy apps, use [top navigation](/components/top-navigation) as it's better suited for providing links to a large number of pages.
+For intranet-style and content-heavy apps, use [top navigation](/components/top-navigation) as it's better suited for providing links to a large number of pages.
```

Comment:
```suggestion
For intranet-style and content '&ndash;' heavy apps, use [top navigation](/components/top-navigation) as it's better suited for providing links to a large number of pages.
```

Can you use the '&ndash;' here ? Just thought to mention for consistency with your previous work I have reviewed.